middleware/index.js

```diff
@@ -0,0 +1,55 @@
+const Campground = require("../models/campground");
+const Comment = require("../models/comment");
+
+const middlewareObj = {};
+
+middlewareObj.checkCampgroundOwnership = function (req, res, next) {
```

Comment:
Hmmm. In real life, putting all middlewares in one file is not a good practice. Soooo, instead of index.js, always create a new file for each “middlewareObj” here. For example: middlewares/checkCampgroundOwnership.js